testing/src/mocking.py

```diff
@@ -371,6 +373,11 @@ def status_set(
         *,
         is_app: bool = False,
     ):
+        if status not in get_args(_SettableStatusName):
+            raise ModelError(
+                f'ERROR invalid status "{status}", expected one of '
+                f"[maintenance blocked waiting active]"
```

Comment:
Nit:

```suggestion
                f'[maintenance blocked waiting active]'
```

It's weird that this passes Ruff-ing, given our quote style.

Reply:
I actually had to change it *to* this, having the single quotes first. I opened https://github.com/canonical/operator/issues/1419 as a result :)

Basically: Scenario used double-quotes and so all the code in `testing/` is expected to follow that pattern (and ruff is configured to know that, in the `testing/pyproject.toml`.

I think it's better to be consistent now (rather than putting in an ignore here) and then do the change in a big bang commit.

Reply:
minor: a bit weirded out by the 2nd string being an f-string without interpolation.
I wonder if the allowed set can be computed rather than hard-coded, making the entire string shorter, fitting on one line, and making this discussion a moot point.

Reply:
I remember when I used pylint it would complain about f-strings without interpolation, but I guess ruff doesn't mind or we have it configured not to?

Reply:
See the [long discussion upstream](https://github.com/astral-sh/ruff/pull/10886) for details, but basically if it's a concatenated string then f is allowed if it's used in any of the parts, because you get nicer (subjectively) layout where the string starts match.

Dima's suggestion to avoid it rather than going one way sounds good to me. I'll look into that.

Reply:
>if it's a concatenated string then f is allowed if it's used in any of the parts

Nice, that's what I always wanted when pylint complained